compute/utils/layers.py

```diff
@@ -152,10 +152,14 @@ def layers_match(layers, ltol=0.2, stol=0.3, angle_tol=5.0):
     # Create an instance of Structure Matcher with the given tolerances
     sm = StructureMatcher(ltol, stol, angle_tol)
     # Translate the refence layer (first) from ASE to pymatgen format
-    ref_layer = adaptor.get_structure(layers[0])
+    ase_reflayer = layers[0].copy()
+    ase_reflayer.cell[2] = np.array([0, 0, 2 * ase_reflayer.cell[2, 2]])
```

Comment:
```suggestion
    # We enforce the third vector to be orthogonal to the planes, and we extend by
    # a small factor the third direction length (not too much as the thresholds are
    # in relative units) to avoid to still have some periodicity (e.g. for systems with
    # 1 layer only)
    ase_reflayer = layers[0].copy()
    ase_reflayer.cell[2] = np.array([0, 0, 2 * ase_reflayer.cell[2, 2]])
```